### ISSUE TITLE
关于使用rangenet的问题 (Problem when using rangenet)

### ISSUE THREAD

**Issue by pikaqiu0131** [PRBonn/LiDAR-MOS#69]
您好，我在使用您的代码时，infer这步出现了如下问题：
Couldn't load backbone, using random weights. Error:  ../../../data/model_rangenet_residual_1/backbone is a zip archive (did you mean to use torch.jit.load()?)
Couldn't load decoder, using random weights. Error:  ../../../data/model_rangenet_residual_1/segmentation_decoder is a zip archive (did you mean to use torch.jit.load()?)
Couldn't load head, using random weights. Error:  ../../../data/model_rangenet_residual_1/segmentation_head is a zip archive (did you mean to use torch.jit.load()?)

Infering in device:  cuda
OMP: Warning #191: Forking a process while a parallel region is active is potentially unsafe.
OMP: Warning #191: Forking a process while a parallel region is active is potentially unsafe.
OMP: Warning #191: Forking a process while a parallel region is active is potentially unsafe.

 
但之后还可以生成prediction文件，并且在评估iou时，08序列res=1时为0.005（这个估计是错误的）您知道怎么回事吗

**Comment by Chen-Xieyuanli** [comment]
感谢使用我们的代码，这应该是torch版本的问题，可以参看rangenet的[安装流程](https://github.com/PRBonn/LiDAR-MOS/blob/main/mos_RangeNet/README.md)或者尝试我们的salsanext版本.

**Comment by pikaqiu0131** [comment]
刚刚看了一下我的torch：
>> import torch
>>> print(torch.__version__)
1.1.0
>>> import torchvision
>>> print(torchvision.__version__)
0.2.2

然后咱们的requirement是：
torch==1.1.0
tensorflow==1.13.1
vispy==0.5.3
torchvision==0.2.2.post3

应该是一致的把

**Comment by Chen-Xieyuanli** [comment]
pretrained模型下载了之后要先解压，从log文件来看，你好像在加载zip文件

**Comment by pikaqiu0131** [comment]
您好，我使用这篇文章https://zhuanlan.zhihu.com/p/454415689，更新了一下pytorch版本，现在已经成功运行，就是运行的时候使用了cpu，您知道怎么切换回cuda吗
model folder exists! Using model from ../../../data/model_rangenet_residual_1
Sequences folder exists! Using sequences from ../../../data/sequences
parsing seq 08
Using 4661 scans from sequences [8]
Using DarknetNet53 Backbone
Depth of backbone input =  6
Original OS:  32
New OS:  32
Strides:  [2, 2, 2, 2, 2]
Decoder original OS:  32
Decoder new OS:  32
Decoder strides:  [2, 2, 2, 2, 2]
Total number of parameters:  50372739
Total number of parameters requires_grad:  50372739
Param encoder  40585792
Param decoder  9786080
Param head  867
Successfully loaded model backbone weights
Successfully loaded model decoder weights
Successfully loaded model head weights
********************************************************************************
Cleaning point-clouds with kNN post-processing
kNN parameters:
knn: 5
search: 5
sigma: 1.0
cutoff: 1.0
nclasses: 3
********************************************************************************
Infering in device:  cpu
Infered seq 08 scan 000000.label in 10.995035409927368 sec
Infered seq 08 scan 000001.label in 9.829725742340088 sec
